.github/workflows/build.yml

```diff
@@ -32,14 +36,20 @@ jobs:
         architecture: 'x86'
     - name: Set up Python 3.8 x64
       uses: actions/setup-python@v2
-      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      if: ${{ matrix.os == 'ubuntu-latest' }}
       with:
         python-version: 3.8
         architecture: 'x64'
+    - name: Set up Python for macOS
+      if: ${{ matrix.os == 'macos-latest' }}
+      run: |
+        curl https://www.python.org/ftp/python/${{ matrix.PYTHON_VERSION }}/python-${{ matrix.PYTHON_VERSION }}-macosx10.9.pkg -o "python.pkg"
```

Comment:
Can we validate the hash of this file after the download? I think `shasum` will already be installed on the Mac agents.

Reply:
Yes, inserted the code to do this.